server/maintenance_requests.go

```diff
@@ -1,34 +1,26 @@
 // This file provides NewApp() with functions for maintenance requests
 package main
 
-import(
-	"github.com/gin-gonic/gin"
-	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
-	"time"
-	"io/ioutil"
-	"net/http"
+import (
 	"encoding/json"
-)
-
-/*type CreateRequest struct {
-	Request string `json:"request"`
-	UserSubmitted int `json:"id"`
-	DateSubmitted time.Time `json:"date"`
-}*/
+	"io/ioutil"
 
-/* AllMaintenanceRequests returns all the maintenance_requests from the database
-*  AllMaintenanceRequests is a function of app */
-func (app *App) AllMaintenanceRequests(c *gin.Context) {
-	/* Get data using psql.AllMaintenanceRequests and pass in app.db for the
-	* connection */
+	"net/http"
+	"time"
 
-	/* Takes the data returned in the psql func and formats it to a valid format
-	* using the gin.H func. See if it you can just use a Struct */
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+	"github.com/gin-gonic/gin"
+)
 
-	/* Use the gin.context.JSON func to return a status and the data. See auth.go
-	* for an example */
+// GetMaintenanceRequests returns all the maintenance_requests from the database
+// GetMainte
+func (app *App) getMaintenanceRequests(c *gin.Context) {
+	requests, _ := app.db.AllMaintenanceRequests()
```

Comment:
we shouldn't just ignore the error here, need to handle it and return an error status code to the user somehow